### PR TITLE
Update rubygem-hammer_cli_foreman_remote_execution to 0.1.2

### PR DESCRIPTION
**Pull request by evgeni**
(cherry picked from commit 32c880db0f7548e0307411fb3bcecf41c873acc4)

### CODE REVIEW

File: packages/plugins/rubygem-hammer_cli_foreman_remote_execution/rubygem-hammer_cli_foreman_remote_execution.spec [hunk 1]
```diff
@@ -1,34 +1,40 @@
...
-Summary: Foreman Remote Execution commands for Hammer CLI
 Name: %{?scl_prefix}rubygem-%{gem_name}
-Version: 0.1.1
+Version: 0.1.2
 Release: 1%{?foremandist}%{?dist}
-Group: Applications/System
-License: GPLv3
+Summary: CLI for the Foreman remote execution plugin
+Group: Development/Languages
+License: GPL-3.0-or-later
```
Comment (ekohl) [inline packages/plugins/rubygem-hammer_cli_foreman_remote_execution/rubygem-hammer_cli_foreman_remote_execution.spec:16]: This isn't a correct RPM license tag

File: licenses.json [hunk 2]
```diff
@@ -2,5 +2,6 @@
   "GPL-2.0-or-later": "GPLv2+",
   "GPL-3.0": "GPLv3",
   "GPL-3.0+": "GPLv3+",
-  "GPLv3-or-later": "GPLv3+"
+  "GPLv3-or-later": "GPLv3+",
```
Comment (ekohl) [inline licenses.json:5]: AFAIK this is not a valid SPDX license code. Should we keep it out of the cherry pick commit?

Comment (evgeni) [reply licenses.json:5]: GPL-3.0-or-later is valid, and that's what I added. GPLv3-or-later is not, but that was here before.

Comment (ekohl) [reply licenses.json:5]: Yes, I later realized this was the cherry pick PR.

**Pull request merged**